engine/includes/components.hpp

```diff
@@ -114,6 +114,10 @@ struct Player{
     Player(int id) : id(id) {}
 };
 
+struct Npc {
+    //TODO
+};
```

Comment:
To do as in used but not defined or future feature?

Reply:
Future use